tock/hours/views.py

```diff
@@ -586,17 +593,11 @@ def user_timeline_view(request):
         timecard__user__user_data__organization__name='organization'
     )
 
+
 @api_view(['GET'])
 def admin_bulk_timecard_list(request):
     if not request.user.is_superuser:
-        return render(
-            request,
-            'uaa_client/login_error.html',
-            context={
-                'superuser': request.user.is_superuser,
-            },
-            status=403
-        )
+        return render_to_response('403.html')
```

Comment:
Is there a reason we're not just raising `PermissionDenied` here, and allowing Django (or DRF, I am not sure which) to take care of the actual rendering?

An interesting aspect of these API endpoints in particular: raising `PermissionDenied` might actually be especially effective, because my hunch is that DRF will render a different response based on whether a programmatic client or a web browser is on the other end, because it has a whole funky [content negotiation](http://www.django-rest-framework.org/api-guide/content-negotiation/) strategy.

Reply:
Oh good call, thanks!